models/spring-ai-elevenlabs/src/main/java/org/springframework/ai/elevenlabs/ElevenLabsTextToSpeechOptions.java

```diff
@@ -0,0 +1,410 @@
+	}
+
+	@Override
+	public boolean equals(Object o) {
+		if (this == o)
+			return true;
+		if (!(o instanceof ElevenLabsTextToSpeechOptions that))
+			return false;
+		return Objects.equals(modelId, that.modelId) && Objects.equals(voiceId, that.voiceId)
+				&& Objects.equals(outputFormat, that.outputFormat) && Objects.equals(voiceSettings, that.voiceSettings)
+				&& Objects.equals(languageCode, that.languageCode)
+				&& Objects.equals(pronunciationDictionaryLocators, that.pronunciationDictionaryLocators)
+				&& Objects.equals(seed, that.seed) && Objects.equals(previousText, that.previousText)
+				&& Objects.equals(nextText, that.nextText)
+				&& Objects.equals(previousRequestIds, that.previousRequestIds)
+				&& Objects.equals(applyTextNormalization, that.applyTextNormalization)
+				&& Objects.equals(nextRequestIds, that.nextRequestIds)
+				&& Objects.equals(applyLanguageTextNormalization, that.applyLanguageTextNormalization);
+	}
+
+	@Override
+	public int hashCode() {
+		return Objects.hash(modelId, voiceId, outputFormat, voiceSettings, languageCode,
+				pronunciationDictionaryLocators, seed, previousText, nextText, previousRequestIds, nextRequestIds,
+				applyTextNormalization, applyLanguageTextNormalization);
+	}
+
+	@Override
+	public String toString() {
+		return "ElevenLabsSpeechOptions{" + "modelId='" + modelId + '\'' + ", voiceId='" + voiceId + '\''
+				+ ", outputFormat='" + outputFormat + '\'' + ", voiceSettings=" + voiceSettings + ", languageCode='"
+				+ languageCode + '\'' + ", pronunciationDictionaryLocators=" + pronunciationDictionaryLocators
+				+ ", seed=" + seed + ", previousText='" + previousText + '\'' + ", nextText='" + nextText + '\''
+				+ ", previousRequestIds=" + previousRequestIds + ", nextRequestIds=" + nextRequestIds
+				+ ", applyTextNormalization=" + applyTextNormalization + ", applyLanguageTextNormalization="
+				+ applyLanguageTextNormalization + '}';
+	}
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public ElevenLabsTextToSpeechOptions copy() {
+		return ElevenLabsTextToSpeechOptions.builder()
+			.modelId(this.getModelId())
+			.voice(this.getVoice())
+			.voiceId(this.getVoiceId())
+			.format(this.getFormat())
+			.outputFormat(this.getOutputFormat())
+			.voiceSettings(this.getVoiceSettings())
+			.languageCode(this.getLanguageCode())
+			.pronunciationDictionaryLocators(this.getPronunciationDictionaryLocators())
+			.seed(this.getSeed())
+			.previousText(this.getPreviousText())
+			.nextText(this.getNextText())
+			.previousRequestIds(this.getPreviousRequestIds())
+			.nextRequestIds(this.getNextRequestIds())
+			.applyTextNormalization(this.getApplyTextNormalization())
+			.applyLanguageTextNormalization(this.getApplyLanguageTextNormalization())
+			.build();
+	}
+
+	public static class Builder {
+
+		private final ElevenLabsTextToSpeechOptions options = new ElevenLabsTextToSpeechOptions();
+
+		public Builder modelId(String modelId) {
```

Comment:
the builder should focus on the portable option name 'model' and not 'modelId'.  The same for prefering 'voice' and not having a builder method for 'voiceId'

Reply:
or the opposite... add a 'model' builder method...  I think I'll go that route in merging the PR and add javadoc to the similar methods saying they do the same things..